medical_center/views/medical_patient.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016-2017 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+	<record id="medical_patient_view_form" model="ir.ui.view">
+		<field name="name">medical.patient.form</field>
+		<field name="model">medical.patient</field>
+		<field name="inherit_id" ref="medical.medical_patient_view_form" />
+		<field name="arch" type="xml">
+			<xpath expr="//field[@name='parent_id']" position="after">
+				<field name="medical_center_primary_id" />
+				<field name="medical_center_secondary_ids"
+				       widget="many2many_tags"
+					   />
+			</xpath>
+		</field>
+	</record>
+
+	<record id="medical_patient_view_search" model="ir.ui.view">
+		<field name="name">medical.patient.search</field>
+		<field name="model">medical.patient</field>
+		<field name="inherit_id" ref="medical.medical_patient_view_search" />
+		<field name="arch" type="xml">
+			<xpath expr="//field[@name='parent_id']" position="after">
+				<field name="medical_center_primary_id" />
+				<field name="medical_center_secondary_ids" />
+			</xpath>
+			<xpath expr="//search" position="inside">
+				<filter string="Primary Medical Center"
+						context="{'group_by': 'medical_center_primary_id'}"
+						/>
+			</xpath>
+		</field>
+	</record>
+</odoo>
```

Comment:
Lint is complaining that you used tabs and not spaces (`Use wrong tabs indentation instead of four spaces`) throughout this file.